CADETPythonSimulator/unit_operation.py

```diff
@@ -180,6 +191,27 @@ def Q_out(self) -> np.ndarray:
 
         return self._Q_out
 
+    @Q_out.setter
+    def Q_out(self, Q_out: npt.ArrayLike):
+        if not (len(Q_out) == self.n_outlet_ports):
+            raise ValueError(
+                f"""Q_out has wrong shape. Expected ({self.n_outlet_ports},),
+                got ({len(Q_out)},)"""
+            )
+        self._Q_out = np.array(Q_out)
+
+    def set_Q_in(self, port: int, Q_in: float):
```

Comment:
Why do we have this method?

If we need it, add a docstring

Reply:
We have this method because it simplyfies the Q_in/Q_out setting during the coupling of unti operations, because the connectifity matrix calculated in `system.connectivity` has portspecific rows, so it is more easy to set the specific rate to a port instead of genareting the whole flow array.

Already added a short docstring, consider renaming it to `set_Q_out_port`?